lib/Installation/Partitioner/LibstorageNG/v4/ExpertPartitionerController.pm

```diff
@@ -115,7 +115,7 @@ sub edit_partition_on_gpt_disk {
     $self->get_expert_partitioner_page()->select_item_in_system_view_table($args->{existing_partition});
     $self->get_expert_partitioner_page()->press_edit_partition_button();
     $self->get_edit_formatting_options_page()->select_format_device_radiobutton();
-    $self->get_edit_formatting_options_page()->select_filesystem($args->{fs_type});
+    $self->get_edit_formatting_options_page()->select_filesystem($args->{fs_type}, $args->{skip});
```

Comment:
Why you didn't skip `select_format_device_radiobutton()`? I had a hard time understanding skip on that point...you want to edit a partition but only changes mounting options. I guess you need to specialize your code for aarch64 and remove that method, if I get correctly the idea of this OOP framework, that skip works kind of like branching (if/else) that should be unnecessary, @OleksandrOrlov WDYT?

Reply:
That is more or less what I asked in the ticket "deal with several partitions and skip the select_filesystem part under a condition". So effectively it makes sense to skip the also select_format_device_radiobutton. Maybe we can find better way. 
My idea was to effectively add something like
```
unless $skip_fs_selection {
  $self->get_edit_formatting_options_page()->select_format_device_radiobutton();
  $self->get_edit_formatting_options_page()->select_filesystem($args->{fs_type});
}
```
But @b10n1k thought we should not put the logic here.
We can also create another method, but it will look almost identical, so that's duplicate code...
If you have a better idea, please propose. I don't really understand why we would need to remove that method, I suppose you meant create a new one rather than remove ?

Reply:
Ahm ok, you're right, format device still needs to be selected (I thought it was selected by default), so it is the default value. I think with this shortcut we are making the test more smart that it should, we are not asserting what is the current value, as in the previous case, in the combobox and if the order would change or more format would be available (I remember something like that in the past) it would fail.

Reply:
What about to pass the parameter for `edit_partition_on_gpt_disk` and not run that line if this flag is set?

Reply:
> What about to pass the parameter for `edit_partition_on_gpt_disk` and not run that line if this flag is set?

Isn't it what I proposed above ? Or do you think of something else ?

Reply:
```
sub edit_partition_on_gpt_disk {
    my ($self, $args) = @_;
    ...
    $self->get_edit_formatting_options_page()->select_filesystem($args->{fs_type}) unless ($args->{skip_filesystem_selection.})
...
```

Reply:
yes, I got confused if we need to select the radiobutton or not, so yes, I like more your proposal using better name for the skip.